functions/__init__.py

```diff
@@ -2,13 +2,24 @@
 import os
 import exiftool
 import sqlite3
-import uuid
 
+class Database:
+    conn = None
 
+    @staticmethod
+    def get_connection():
+        if Database.conn is None:
+            Database.conn = sqlite3.connect('uploads/database.db')
+        return Database.conn
 
+    @staticmethod
+    def close_connection():
+        if Database.conn is not None:
+            Database.conn.close()
+            Database.conn = None
```

Comment:
This is fine as long as there is no multithreading. If the `Database` class can be accessed from several threads, here there are race conditions on every check of whether `Database.conn` is `None`.

In general, I don't think it's the best way to organize database connection management but, as long as we're using sqlite and there really isn't multi-threaded connections, I think it's fine for this project